src/robot/libraries/BuiltIn.py

```diff
@@ -2220,8 +2226,14 @@ def wait_until_keyword_succeeds(self, retry, retry_interval, name, *args):
             if count <= 0:
                 raise ValueError('Retry count %d is not positive.' % count)
             message = '%d time%s' % (count, s(count))
+        strict_interval = isinstance(retry_interval, str) \
+            and retry_interval.replace(' ', '').lower().startswith('strict:')
```

Comment:
Need to use our `is_string` helper here due to us still for a little time supporting Python 2 where `str` is actually byes.